<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OfferSearch</class>
 <widget class="QWidget" name="OfferSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OfferSearch</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_filtruj">
     <property name="text">
      <string>Filtruj:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_id">
      <attribute name="title">
       <string>Symbol oferty</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEdit_Id"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_klient">
      <attribute name="title">
       <string>Klient</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="lineEdit_Client"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_date">
      <attribute name="title">
       <string>Data</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QDateEdit" name="dateEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_user">
      <attribute name="title">
       <string>Oferent</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="comboBox"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_wybierz">
     <property name="text">
      <string>Wybierz ofertÄ™ do wczytania:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
